src/components/BShuttle/BShuttle.vue

```diff
@@ -35,7 +35,7 @@
           role='option'
           tabindex='0'
         >
-        {{ option.displayText }}
+          {{ option.value }}
```

Comment:
This should be option.displayText

Reply:
Radio and checkbox arrays all use id and value, figured it should be consistent

Reply:
oh I see what you did.  Im fine with that.. but technically the text is not the value of the option in a traditional sense, the Id is.  That's why I didn't use value, it may cause confusion.

Reply:
Select was also updated to reflect the use if id, value.  Do we need to have a mini retro to discuss what is best.  When @fobesjason suggested it I thought it made sense.  Not having to keep track of displayName vs Value for different components.  They all return ID anyway.